docs/guides-document-approval.md

```diff
@@ -485,11 +488,14 @@ namespace Elsa.Guides.DocumentApproval.WebApp
         public void ConfigureServices(IServiceCollection services)
         {
             services
-                .AddElsa()
```

Comment:
Actually, `.AddElsa()` is still supported and the most convenient way to install Elsa into DI. `.AddElsaCore()` also works, but requires the developer to register additional services manually (if they want to use them).